components/gitpod-protocol/src/installation-admin-protocol.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+export interface InstallationAdminSettings {
+    sendTelemetry: boolean;
+}
+
+export interface InstallationAdmin {
+    id: string;
+    settings: InstallationAdminSettings;
```

Comment:
If we made this `sendTelemetry?: boolean,` we could avoid settings an explicit default (as long as it's `false`, that is).

Also, :+1: for using an interface (if you forsee additional fields):
```
export interface Telemetry {
    enabled: boolean,
    // userStatistics: boolean,
    // traceEvenets: boolean,
}
```

Reply:
My expectation is that `sendTelemetry` will always be populated - if the record doesn't exist in the DB, it's created for us and the parameter is set.

Unless you strongly favour an optional flag, I'd suggest keeping it as required

Reply:
There definitely will be additional fields too

Reply:
> My expectation is that sendTelemetry will always be populated - if the record doesn't exist in the DB

Got that. My point here ws that we should try hard to minimize the code changes required here. ATM one could argue that `server`/`gitpod-db` don't need to now _anything_ about the internal structure of the `InstallationAdminSettings` shape.
But maybe this is just starting yesterdays discussion all over again.

Reply:
I guess I'm in your hands as you're the owners of this component. Realistically, I'm happy to do anything you want - I don't even mind if you wanted to do the changes in your own team although I've proposed doing them in my team as I know how much work you have on.

So long as we have a generated ID that remains the same per-installation, and a `sendTelemetry` boolean that we can update, I don't mind how you want me to do it.

Reply:
> So long as we have a generated ID that remains the same per-installation, and a sendTelemetry boolean that we can update, I don't mind how you want me to do it.

Alright. Current code LGTM :slightly_smiling_face: